docs/backup-restore.md

```diff
@@ -23,3 +23,10 @@ One requirement for backup in Scalar DB on JDBC databases is that backups for al
 That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
 Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
+
+## Scalar DB server
+
+Since the Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, we can easily pause/unpause it as mentioned in [Cassandra](#Cassandra). 
```

Comment:
```suggestion
Since Scalar DB server implements [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface, you can easily pause the server to stop accepting incoming requests and drain in-flight requests.
```